### PR TITLE
Make core selection githuby in reusable-integration-tests

### PR DESCRIPTION
**Pull request by szepeviktor**
GitHub Actions has an `if` step condition.

### CODE REVIEW

File: .github/workflows/reusable-integration-tests.yml [hunk 1]
```diff
@@ -52,21 +52,24 @@ jobs:
         key: ${{ inputs.php }}-composer-${{ hashFiles('composer.json') }}
 
     - name: Debugging
+      shell: 'bash -x {0}'
```
Comment (szepeviktor) [inline .github/workflows/reusable-integration-tests.yml:55]: I've opened its output last time and was unable to distinguish the four outputs.

File: .github/workflows/reusable-integration-tests.yml [hunk 2]
```diff
@@ -52,21 +52,24 @@ jobs:
...
 
     - name: Install Composer dependencies
-      run: |
-        if [ "${{ inputs.wp }}" == "latest" ]; then
-          composer install --dev --prefer-dist
-        elif [ "${{ inputs.wp }}" == "nightly" ]; then
-          composer require --dev --update-with-dependencies --prefer-dist roots/wordpress-full="dev-main"
-        else
-          composer require --dev --update-with-dependencies --prefer-dist roots/wordpress-full="~${{ inputs.wp }}.0"
-        fi
```
Comment (szepeviktor) [inline .github/workflows/reusable-integration-tests.yml:69]: No more shell code.

**Comment by szepeviktor** [comment]
If you merge this one I may continue with other reusable workflows.

**Comment by szepeviktor** [comment]
How about adding a workflow to this repository that tests reusable workflows ❔

**Comment by szepeviktor** [comment]
@johnbillion Maybe 🖱️ a click here too?

**Comment by johnbillion** [comment]
I'm going to leave this one Viktor because I think I might end up introducing some more conditional logic, and the GitHub actions syntax doesn't allow for regex matching or anything very complicated, and I think I'll need it.

Thanks anyway!

**Comment by szepeviktor** [comment]
All right. I'll wait for that _anything very complicated_  :)

**Pull request closed without merging**